LeaderboardBackend/Services/Impl/LeaderboardService.cs

```diff
@@ -12,17 +12,14 @@ public LeaderboardService(ApplicationContext applicationContext)
         _applicationContext = applicationContext;
     }
 
-    public async Task<Leaderboard?> GetLeaderboard(long id)
-    {
-        return await _applicationContext.Leaderboards.FindAsync(id);
-    }
+    public async Task<Leaderboard?> GetLeaderboard(long id) =>
+        await _applicationContext.Leaderboards
+            .FirstOrDefaultAsync(board => board.Id == id);
```

Comment:
What's wrong with `FindAsync`? It's more a more concise way to write the same thing, isn't it?

Reply:
I just wasn't aware of it, yeah